package.json

```diff
@@ -80,7 +80,7 @@
     "broccoli-merge-trees": "^1.0.0",
     "broccoli-test-helper": "^1.1.0",
     "calculate-cache-key-for-tree": "^1.0.0",
-    "ember-asset-loader": "^0.3.0",
+    "ember-asset-loader": "git+https://github.com/BBVAEngineering/ember-asset-loader.git#prevent-caching-errors",
```

Comment:
Just leaving a note here so we don't forget to update this to a released version once the other PR is merged

Reply:
This can be updated now.